app/src/main/java/org/jellyfin/androidtv/preference/UserPreferences.kt

```diff
@@ -100,6 +101,11 @@ class UserPreferences(context: Context) : SharedPreferenceStore(
 		 */
 		var refreshRateSwitchingBehavior = enumPreference("refresh_rate_switching_behavior", RefreshRateSwitchingBehavior.DISABLED)
 
+		/**
+		 * Enable 4k codecs for direct play and transcode target
+		 */
+		var enable4kSupport = booleanPreference("pref_enable_4k_support", DeviceUtils.has4kVideoSupport())
```

Comment:
Preference keys should not start with `pref_`. This is only used on old keys for compatibility reasons (creating a migration just for a rename of the key is kinda pointless)